test/models/setting_test.rb

```diff
@@ -63,4 +63,9 @@ def test_class_methods_are_shorthand_for_values
 
     assert_equal(10, Setting.wishlist_size)
   end
+
+  def test_multilocale_violations_count_regulations_link
+    Setting.find_by(code: :violations_count_regulations_link).update!(value: "{\"en\":\"https://example.com#some_anchor\", \"et\":\"https://example.et#some_anchor\"}")
```

Comment:
It seems `et` key has no effect on the test. Can we remove it to avoid one having such assumption?

Also, how about implying value `https://example.com#some_anchor` to say `https://regulations.test`, extracting it to a variable `regulations_link` and reusing it, making a test more readable? (you clearly see the connection between input and output).

Reply:
I'd rather keep that key - it helps us understand that not only hash is saved correctly, but we are getting correct key/value pair from hash.
But changing value to more info is a good idea.

Reply:
> but we are getting correct key/value pair from hash.

Where is it tested, I don't get. Isn't it a concern of unit test, not system?